fpm/build.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+bundle install --path "${HOME}/bundles/${JOB_NAME}"
+
+cd "recipes/${RECIPE_NAME}/"
+
+bundle exec fpm-cook --target deb
+
+rsync -v pkg/*.deb deploy@apt-1.management:/tmp
+
```

Comment:
I feel like the task to transfer the `.deb` package shouldn't be coupled to the task to build the package.

Should we just retain the build artefact, similar to what we do for [govuk_crawler_worker](https://ci.dev.publishing.service.gov.uk/job/govuk_crawler_worker/configure)? We can then create a separate task to add it to Aptly - we need to do that anyway as-is.